odoo/cli/command.py

```diff
@@ -1,17 +1,18 @@
 import argparse
 import contextlib
-import logging
+import re
 import sys
 from inspect import cleandoc
 from pathlib import Path
 
-import odoo
-from odoo.modules import get_module_path, get_modules, initialize_sys_path
+import odoo.cli
+from odoo.modules import initialize_sys_path, load_script
 
-commands = {}
-"""All loaded commands"""
 
+COMMAND_NAME_RE = re.compile(r'^[a-z][a-z0-9\-_]*$', re.I)
```

Comment:
I am just thinking about "-" which is not valid as a python module name. Some cli tools handle "-" in the command as "_" in the module. Not sure about what to do with this.